frontend/pages/campaigns/[campaignId]/index.tsx

```diff
@@ -39,56 +43,74 @@ export default function CampaignDetail() {
   const numTotalCoupons = campaign.promisedAmount / campaign.couponDenomination;
   const numCouponsRedeemed = campaign.donations.primaryDonation.amount / campaign.couponDenomination;
 
+  const statsCardsProps: StatsCardProps[] = [
+    {
+      title: `$${campaign.promisedAmount}`,
+      children: (
+        <Typography variant="h4">
+          will be donated to <strong>{campaign.charities.length} charities</strong> when the campaign ends. This was
+          sponsored by {campaign.primaryDonor.name}.
```

Comment:
Can you include the primary donor avatar and see how it looks? For more credibility

Reply:
No its too small for anyone to see it properly.